src/Root.cpp

```diff
@@ -306,9 +305,6 @@ void cRoot::Start(std::unique_ptr<cSettingsRepositoryInterface> a_OverridesRepo)
 	delete m_FurnaceRecipe;   m_FurnaceRecipe = nullptr;
 	delete m_CraftingRecipes; m_CraftingRecipes = nullptr;
 
-	LOG("Unloading worlds...");
-	UnloadWorlds();
```

Comment:
Are you sure this isn't needed any more? Yes, the `std::map` will delete the worlds but the ordering could be important.

Reply:
indeed, thanks!